context/app/static/js/shared-styles/buttons/index.tsx

```diff
@@ -19,10 +20,17 @@ const svgStyles = {
   fontSize: '1.25rem',
 };
 
-const WhiteBackgroundIconButton = styled(IconButton)<IconButtonProps>({
+const WhiteBackgroundIconButton = styled(IconButton)<IconButtonProps>(({ theme }) => ({
   ...whiteBackgroundCSS,
   '& svg': svgStyles,
-});
+  border: `1px solid ${theme.palette.divider}`,
```

Comment:
You can pull this out into const ``border: `1px solid ${theme.palette.divider}`;``. And then spread it, `...border`, in each case.